core/error.go

```diff
@@ -6,27 +6,27 @@ import (
 )
 
 const (
-	// ResponseReadError occues when error happened while we tried to
+	// ResponseReadError occurs when error happened while we tried to
```

Comment:
How could this pass on lint check as it checks for spelling errors in comments too. 😮